homeassistant/components/konnected/strings.json

```
@@ -18,11 +18,11 @@
       }
     },
     "error": {
-      "cannot_connect": "Unable to connect to a Konnected Panel at {host}:{port}"
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%] {host}:{port}"
```

No need to add host and port here, since we already fill in the host and port that was used in the form.
```suggestion
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
```